docs/guides/construct-circuits.ipynb

```diff
@@ -23,7 +23,7 @@
    "source": [
```

Comment:
<p>I would suggest changing this first couple sentences to:</p><p> </p><p>Measurements are used to sample the states of individual qubits and transfer the results to a classical register. Note that if you are submitting circuits to a <a href="https://docs.quantum.ibm.com/guides/primitives#sampler" target="_blank">Sampler primitive</a>, measurements are required. However, circuits submitted to an <a href="https://docs.quantum.ibm.com/guides/primitives#estimator" target="_blank">Estimator primitive</a> <em>must not</em> contain measurements.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1601/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>